collectors/wallet_collector.go

```diff
@@ -169,4 +183,30 @@ func (u *WalletCollector) Collect(ch chan<- prometheus.Metric) {
 		u.unconfirmedBalanceDesc, prometheus.GaugeValue,
 		float64(walletBal.Unconfirmed),
 	)
+
+	accounts, err := u.lnd.WalletKit.ListAccounts(context.Background(), "", 0)
+	if err != nil {
+		u.errChan <- fmt.Errorf("WalletCollector ListAccounts"+
+			"failed with: %v", err)
+		return
+	}
+
+	for _, account := range accounts {
+		name, addrType, path := account.GetName(),
+			account.GetAddressType().String(), account.GetDerivationPath()
```

Comment:
nit: maybe just define on separate lines so that we keep the 80 char line limit

Reply:
fixed